src/voxel.cpp

```diff
@@ -277,46 +259,38 @@ void VoxelManipulator::copyTo(MapNode *dst, const VoxelArea& dst_area,
 	-----------------------------------------------------
 */
 
-void VoxelManipulator::clearFlag(u8 flags)
+void VoxelManipulator::setFlag(const VoxelArea &a, u8 flags)
```

Comment:
`setFlags` would be more precise. Feel free to ignore if it's not worth the effort.

Reply:
:shrug: